Examples/Tests/reduced_diags_tools/inputs_3d

```diff
@@ -0,0 +1,100 @@
+# Maximum number of time steps
```

Comment:
Rather than having separate input file, could these tests be moved into the `Examples/Tests/reduced_diags` directory and use the existing input file there?

Looking at this a bit more, perhaps the file `Examples/Tests/reduced_diags/analysis_reduced_diags_impl.py` could be modified to use the new tools rather than `np.genfromtxt`?